samples/app-anonymous-users/nodejs/api/server/.env

```diff
@@ -0,0 +1,6 @@
+MicrosoftAppId='{{Microsoft-App-id}}'
+MicrosoftAppPassword='{{MicrosoftAppPassword}}'
+MicrosoftAppTenantId='{{TenantId}}'
+FaceBookAppId="{{FacebookAppId}}"
+FaceBookAppPassword="{{Facebook App Password}}"
+ApplicationBaseUrl="{{domain-name}}"
```

Comment:
Verify if you have added details in readme regarding steps how to get these.

Reply:
Yes i have added